docs/engine/rsonpath/index.md

```diff
@@ -0,0 +1,132 @@
+(rsonpath-guide)=
+# rsonpath
+
+Yet another rsonpath cheat sheet.
+
+## Introduction
+
+[rsonpath] is a blazing fast JSONPath query engine written in Rust.
+It allows to run structure-aware JSONPath queries on JSONs
+as easily as one would query a string with regular expressions.
+[rsonpy] is a lightweight Python wrapper around the Rust library.
+
+To learn fundamentals about rsonpath, please refer to the [rsonbook],
+and specifically the [JSONPath reference].
+
+:::{rubric} jqlang vs. rsonpath
+:::
+
+If you are asking what's wrong with `jq`, the authors provide
+concise answers per [Why choose rsonpath?]:
+
+<blockquote>
+
+The most popular tool for working with JSON data is [jq], but it has
+a few shortcomings:
+
+- it is _extremely_ slow
+- it has a _massive_ memory overhead
+- its query language is non-standard.
+
+To be clear, `jq` is a great and well-tested tool, and `rq` does not
+directly compete with it. If one could describe `jq` as a “sed or awk
+for JSON”, then `rq` would be a “grep for JSON”: It does not allow you
+to slice and reorganize JSON data like `jq`, but instead outclasses it
+on the filtering and querying applications.
+</blockquote>
+
+:::{rubric} Use cases
+:::
+
+<blockquote>
+Rsonpath is made to filter document ahead of parsing/loading. Think of
+it like a way to select a small part of the documents before
+loading it into memory. It will shine in situations where you have a
+JSON file too big to hold in memory, but you nevertheless want to load
+parts of it.
+</blockquote>
```

Comment:
@charles-paperman: I've added fragments of your explanations from the other conversation to the [documentation page](https://tikray--35.org.readthedocs.build/engine/rsonpath/) per c022b06d84. Let me know if you want to see any changes or additions.